src/Fantomas/Daemon.fs

```diff
@@ -64,12 +64,15 @@ type FantomasDaemon(sender: Stream, reader: Stream) as this =
 
                 try
                     let! formatResponse =
-                        CodeFormatter.FormatDocumentAsync(
-                            request.IsSignatureFile,
-                            request.SourceCode,
-                            config,
-                            ?cursor = cursor
-                        )
+                        match cursor with
```

Comment:
Could you swap the order, please?
In this case, I prefer to have the shortest case first.